src/bundle/Resources/public/js/scripts/core/collapse.js

```diff
@@ -1,11 +1,123 @@
-(function (global, doc) {
+(function (global, doc, bootstrap) {
+    let toggleAllTimeout;
+    const MULTI_COLLAPSE_BTN_TAG = 'data-multi-collapse-btn-id';
+    const COLLAPSE_SECTION_BODY_TAG = 'data-multi-collapse-body';
+    const MULTI_COLLAPSE_ELEMENT_SELECTOR = '.ibexa-multicollapse--item';
```

Comment:
We no longer use constants for CSS selectors. We did in the past but they were less convenient compared to just using them directly in the code.